pkg/cli/admin/release/release.go

```diff
@@ -20,9 +24,19 @@ func NewCmd(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Comm
 			registries.
 			`),
 	}
+	cmd.PersistentFlags().StringVar(&GraphArch, "graph-arch", "", "Specify the architecture to find in the release graph (amd64|arm64|ppc64le|s390x) when not connected to a cluster")
```

Comment:
One thing I'm having trouble figuring out.. I want to be able to see the `--graph-arch` option when you run `oc adm release info|extract|mirror --he